.travis.yml

```diff
@@ -8,10 +8,6 @@ node_js:
 
 sudo: false
 
-cache:
```

Comment:
@sandersky - I have also felt this pain and definitely understand the goal/intent here, but I think this addon should try to stick as close to the default addon blueprint as possible.  Straying from that makes future upgrades harder...